.gitignore

```diff
@@ -1 +1,8 @@
 tests/*/results.json
+tests/anagram-all-fail/anagram-all-fail
+tests/anagram-partial-fail/anagram-partial-fail
+tests/anagram-success/anagram-success
```

Comment:
What do these ignores do?

Reply:
The executable are generated there when I was testing.